app/backend/approaches/approach.py

```diff
@@ -53,9 +53,10 @@ class Document:
     reranker_score: Optional[float] = None
 
     def serialize_for_results(self) -> dict[str, Any]:
-        return {
+        result_dict = {
             "id": self.id,
             "content": self.content,
+            # Should we rename to its actual field name in the index?
```

Comment:
This is a pending question as to whether to send it down using the field of the index?

Reply:
Don't share the embedding in the frontend